cargo-dist/src/tasks.rs

```diff
@@ -1399,6 +1421,33 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         checksum_idx
     }
 
+    fn add_updater(&mut self, variant_idx: ReleaseVariantIdx) {
+        if !self.local_artifacts_enabled() {
+            return;
+        }
+
+        let artifact = self.make_updater_for_variant(variant_idx);
+
+        self.add_local_artifact(variant_idx, artifact);
+    }
```

Comment:
might be worth adding a comment here being like "yep this makes an updater for every app, it could be shared between apps, but for an mvp it's fine".